tests/deployment/switch_config.py

```diff
@@ -54,7 +61,30 @@ def server_init():
                                      'site_layout')
 
 
-class TestNativeNetwork(NetworkTest):
+@pytest.fixture
+def not_dell():
+    """open the site-layout file to see if we don't have a dell switch"""
+
+    layout_data = open('site-layout.json')
+    layout = json.load(layout_data)
+    layout_data.close()
+    for switch in layout['switches']:
+        return switch['type'] != DELL
```

Comment:
I wasn't sure if we are ever going to run deployment tests with multiple switches described in the site-layout file. So what I wanted to do here is, assuming we have more than 1 switch, return false if any of the switch isn't a Dell so that we skip that test.